src/main/java/com/machine/learning/classifier/Classifier.java

```diff
@@ -0,0 +1,21 @@
+package com.machine.learning.classifier;
+
+import java.util.List;
+
+public interface Classifier {
+    /**
+     * Trains the classifier on the given data
+     *
+     * @param dataPoints data to train with
+     * @param classLabel type of the class for the data
+     */
+    void train(List<List<Integer>> dataPoints, ClassLabel classLabel);
+
+    /**
+     * Classifies an unknown datapoint using its training
+     *
+     * @param dataPoint data to classify
+     * @return  class label
+     */
+    ClassLabel classify(Integer dataPoint);
```

Comment:
Do you know how classification machine learning works?

For example I might train on several point like:
`1,2,3,Y` with the last field being the class label
and then later, should be able to take a point like:
`1,2,3` and produce the class label `Y` (although it's just memorizing, in this specific case)